quiz.js

```diff
@@ -1,16 +1,18 @@
 
-var growTree = function() {
-  var charT = document.getElementById('char').value;
-  var treeHeight = document.getElementById('height').value;
+var growTree = function(final) {
+  charT = document.getElementById('char').value;
+  treeHeight = document.getElementById('height').value;
 
-  (function () {
+   if (charT === '' || treeHeight === ''){
+        alert('must enter value');
+    }
+
+
+  (function final() {
```

Comment:
Since this is the function that grows your tree there should be an object passed into it.